MOBILE_API.xml

```diff
@@ -3699,7 +3688,428 @@
+        </element>
+        <element name="EXIT"/>
+        <element name="STAY"/>    
+        <element name="MERGE"/>
+        <element name="FERRY"/>
+        <element name="CAR_SHUTTLE_TRAIN"/>
+        <element name="WAYPOINT"/>
+    </enum>
+
+    <enum name="NavigationJunction">
+        <element name="REGULAR">
+            <description> A junction that represents a standard intersection with a single road crossing another. </description>
+        </element>
+        <element name="BIFURCATION">
+            <description> A junction where the road splits off into two paths; a fork in the road. </description>
+        </element>
+        <element name="MULTI_CARRIAGEWAY">
+            <description> A junction that has multiple intersections and paths. </description>
+        </element>
+        <element name="ROUNDABOUT">
+            <description> A junction where traffic moves in a single direction around a central, non-traversable point to reach one of the connecting roads. </description>
+        </element>
+        <element name="TRAVERSABLE_ROUNDABOUT">
+            <description> Similar to a roundabout, however the center of the roundabout is fully traversable. Also known as a mini-roundabout. </description>
+        </element>
+        <element name="JUGHANDLE">
+            <description> A junction where lefts diverge to the right, then curve to the left, converting a left turn to a crossing maneuver. </description>
+        </element>
+        <element name="ALL_WAY_YIELD">
+            <description> Multiple way intersection that allows traffic to flow based on priority; most commonly right of way and first in, first out. </description>
+        </element>
+        <element name="TURN_AROUND">
+            <description> A junction designated for traffic turn arounds. </description>
+        </element>
+    </enum>
+
+    <enum name="Direction">
+        <element name="LEFT"/>
+        <element name="RIGHT"/>
+    </enum>
+
+    <struct name="NavigationInstruction">
+        <param name="locationDetails" type="LocationDetails" mandatory="true"/>
+
+        <param name="action" type="NavigationAction" mandatory="true"/>
+
+        <param name="eta" type="DateTime" mandatory="false"/>
+
+        <param name="bearing" type="Integer" minvalue="0" maxvalue="359" mandatory="false">
+            <description> The angle at which this instruction takes place. For example, 0 would mean straight, less than 45 is bearing right, greater than 135 is sharp right, between 45 and 135 is a regular right, and 180 is a U-Turn, etc. </description>
+        </param>
+
+        <param name="junctionType" type="NavigationJunction" mandatory="false"/>
+
+        <param name="drivingSide" type="Direction" mandatory="false">
+            <description> Used to infer which side of the road this instruction takes place. For a U-Turn (Action=Turn, direction=180) this will determine which direction the turn should take place. </description>
```

Comment:
* _Action=Turn_ should be formatted like so: _action=TURN_
* _direction=180_ should be _bearing=180_

Reply:
Fixed in https://github.com/smartdevicelink/rpc_spec/pull/149/commits/06a3394c636aa24d9b880a352c67c94700476e0e